tasks/system_probe.py

```diff
@@ -1527,24 +1527,26 @@ def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
 @task
 def test_microvms(
     ctx,
-    security_groups=None,
-    subnets=None,
+    infra_env,
     instance_type_x86=None,
     instance_type_arm=None,
     x86_ami_id=None,
     arm_ami_id=None,
     destroy=False,
     upload_dependencies=False,
+    ssh_key_name=None,
+    ssh_key_path=None,
 ):
     args = [
-        f"--sgs {security_groups}" if security_groups is not None else "",
-        f"--subnets {subnets}" if subnets is not None else "",
         f"--instance-type-x86 {instance_type_x86}" if instance_type_x86 is not None else "",
         f"--instance-type-arm {instance_type_arm}" if instance_type_arm is not None else "",
         f"--x86-ami-id {x86_ami_id}" if x86_ami_id is not None else "",
         f"--arm-ami-id {arm_ami_id}" if arm_ami_id is not None else "",
         "--destroy" if destroy else "",
         "--upload-dependencies" if upload_dependencies else "",
+        f"--ssh-key-path {ssh_key_path}" if ssh_key_path is not None else "",
```

Comment:
I *think* this could be simplified to this:
```suggestion
        f"--ssh-key-path {ssh_key_path}" if ssh_key_path else "",
```